telemetry/definitions/commonDefinitions.json

```diff
@@ -209,6 +209,11 @@
             ],
             "description": "User inputted check type to denote which custom check to run."
         },
+        {
+            "name": "clientLatency",
```

Comment:
Since teams have indicated that it's hard to find related metrics, should we deviate a bit from our normal pattern and use `latency` as a prefix for all of this? Or maybe even `perf` as a prefix would help group all perf-related fields in the future.

Reply:
I think things like latency and duration are already naturally searchable. What advantages would we get if we prefixed with latency or perf? My problem with telemetry right now is every time I search for something that could be general it starts with the prefix of a team/project name 😅

Reply:
they will be grouped together